app/assets/ens/register.js

```diff
@@ -1,18 +1,22 @@
 window.onload = function() {
-  $('#register')[0].onclick = function(e) {
-    if ($('#tos')[0].checked === false) {
-      return _alert({ message: gettext('Please check the TOS checkbox.')}, 'error');
-    }
-    if (typeof web3 == 'undefined') {
-      return _alert({ message: gettext('Please install metamask in your browser and try again.')}, 'error');
-    }
-    // Get the github handle
-    githubHandle = $('#githubHandle').text();
-    // Get the current account and call sign function:
-    web3.eth.getAccounts(function(err, accounts) {
-      if (!accounts)
-        return;
-      signMsgAndCreateSubdomain('Github Username : ' + $('#githubHandle').text(), accounts[0]);
-    });
-  };
+  var registerBtn = $('#register');
```

Comment:
Whats up with these changes to the ENS code?

Reply:
This was another bug. Register is ENS domain name register button. Result jQuery returns is never checked and attempt of taking 0th items from empty may fail with javascript error. Fixed that also.

Reply:
Ah ok - nice! Thanks for the fix! Can you update the PR title and description with this info? Also, 
 were you able to test the ENS fix locally?

Reply:
Will update title and description. As I stated twice - I've tested ENS locally. It works my my Raven fix.

Reply:
Apologies - I don't see your mention of how you tested the ENS fix - I see in https://github.com/gitcoinco/web/issues/3981 that you mention you'd make a video but I don't see that either - can you point me to where you mentioned how it was tested? Thanks!

Reply:
I've said it been tested in source code comment (1st time), you saw that. 2nd time I change this issue description and also mentioned which version of Opera I used to test it and make sure it works. I haven't make the video as I don't wanna spend another hour installing and setting up video capturing software being waiting for 3 months my $50 work to be reviewed. Sorry for that.

Reply:
Hey @agutsal actually you said you were going to record it here 24 days ago  https://github.com/gitcoinco/web/issues/3981#issuecomment-500238507 
We were waiting for it since we got a lot of PRs we can't check locally every PR, so is that why part of the task is always show some test or something that demonstrate is working as it should if you have problem with video capture you can use https://chrome.google.com/webstore/detail/screen-and-webcam-recorde/jiihcciniecimeajcniapbngjjbonjan is not going to take you more than 1 min installing it.

Reply:
Well, I though you wanna me to screen capture shell+browser that shows upgrading from master to my PR commit. But if you want just browser - that's OK.
Drop me few cents in ETH to 0x6c7623Eed8fb55dE471B3AfA2f44afcC2e2946d4 so I can register new ENS name and show you.